src/Controller/SubjectsController.php

```diff
@@ -138,6 +142,31 @@ private function createFirstPage(array $arguments) : Response
             }))
             ->otherwise($this->softFailure('Failed to load highlights for '.$arguments['id']));
 
+        $arguments['seniorEditors'] = (new PromiseSequence($this->get('elife.api_sdk.people')
+            ->forType('senior-editor')
+            ->forSubject($arguments['id'])
+            ->reverse()
+            ->slice(0, 100)))
+            ->sort(Callback::call('rand', 0, 1))
```

Comment:
Weird to have randomness in a page but ok (caches will rotate it more slowly than expected)